internal/util/version/version.go

```diff
@@ -40,6 +40,12 @@ var (
 	branch string
 )
 
+// Fake MongoDB version for clients that check it.
+var (
+	MongoDBVersion      = "5.0.42"
```

Comment:
How is possible to return when in the docker-compose file the version is `mongo:5.0.8`?

Reply:
.42 is obviously a number that nobody will parse seriously.